### PR TITLE
feat: Add preconfigured support for Smart Agent

### PR DESCRIPTION
**Pull request by jtmal-signalfx**

### CODE REVIEW

**Review by theletterf — DISMISSED** [review]

**Comment by jtmal-signalfx** [comment]
> The issue where exporters set with the environment variable can't be external also exists for the propagators. The argument was that it is only the environment variable and you are suppose to set custom propagators through the configuration setup code. Is that possible for exporters as well? If not that may be what we want to contribute back.

Sent <https://github.com/open-telemetry/opentelemetry-ruby/issues/1365>

File: README.md [hunk 1]
```diff
@@ -52,8 +51,15 @@ propagation format set `OTEL_PROPAGATORS` to `b3multi`:
 export OTEL_PROPAGATORS=b3multi
 ```
 
-# License
+## Configure for use with Smart Agent
+
+And exporter called `jaeger-thrift-splunk` is provided and preconfigured to send data to locally (localhost) running instance of [Smart Agent](https://github.com/signalfx/signalfx-agent).
```
[the remark this answers is not in the document]
Comment (jtmal-signalfx) [reply README.md:222]: https://github.com/signalfx/splunk-otel-ruby/blob/main/splunk-otel.gemspec#L29, right?

**Comment by theletterf** [comment]
There's still a docs suggestion to merge

**Review by tsloughter — APPROVED** [review]

**Review by tsloughter-splunk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,8 +51,15 @@ propagation format set `OTEL_PROPAGATORS` to `b3multi`:
...
 
+To use it, list it in an environmental variable `OTEL_TRACES_EXPORTER`, e.g. `OTEL_TRACES_EXPORTER=jaeger-thrift-splunk` or `OTEL_TRACES_EXPORTER=otlp,jaeger-thrift-splunk`.
+
+If environmental variable `SPLUNK_REALM` or `OTEL_EXPORTER_JAEGER_ENDPOINT` is set, then the default endpoint mentioned above will be overwritten.
+
+# License
```
[the remark this answers is not in the document]
Comment (jtmal-signalfx) [reply README.md]: I included this manually while rebasing locally.

**Pull request merged**